src/rai_bench/rai_bench/tool_calling_agent/tasks/custom_interfaces.py

```
@@ -55,799 +53,67 @@
-}
+
 
 SERVICE_STRINGS = [
     f"service: {service}\ntype: {msg_type}\n"
     for service, msg_type in SERVICES_AND_TYPES.items()
 ]
 
 
-PROACTIVE_ROS2_EXPERT_SYSTEM_PROMPT = """You are a ROS 2 expert that want to solve tasks. You have access to various tools that allow you to query the ROS 2 system.
-Be proactive and use the tools to answer questions.
+PROACTIVE_ROS2_EXPERT_SYSTEM_PROMPT_0_SHOT = """You are a ROS 2 expert that want to solve tasks. You have access to various tools that allow you to query the ROS 2 system.
```

Flake8 (7.2.0)

[error] line too long (171 > 79 characters)

(E501)